_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -713,7 +720,41 @@ public static function confirm_safe_mode() {
 				)
 			);
 		}
-		return new WP_Error( 'error_confirming_safe_mode', esc_html__( 'Jetpack could not confirm safe mode.', 'jetpack' ), array( 'status' => 500 ) );
+		return new WP_Error(
+			'error_setting_jetpack_safe_mode',
+			esc_html__( 'Could not confirm safe mode.', 'jetpack' ), array( 'status' => 500 )
+		);
+	}
+
+	/**
+	 * Handles identity crisis mitigation, migrating stats and subscribers from old url to this, new url.
+	 *
+	 * @since 4.4.0
+	 *
+	 * @return bool | WP_Error True if option is properly set.
+	 */
+	public static function migrate_stats_and_subscribers() {
+		$deleted = Jetpack_Options::delete_option( 'sync_error_idc' );
+
+		if ( ! $deleted ) {
+			return new WP_Error(
+				'error_deleting_sync_error_idc',
+				esc_html__( 'Could not delete sync error option.', 'jetpack' ), array( 'status' => 500 )
+			);
+		}
+
+		$updated = Jetpack_Options::update_option( 'migrate_for_idc', true );
+		if ( $updated ) {
+			return rest_ensure_response(
+				array(
+					'code' => 'success'
+				)
+			);
+		}
```

Comment:
In this block, we should also clear the callable await transient probably.

`wp_schedule_single_event( 'jetpack_sync_unlock_sync_callable', time() +1 )`;

Or, we can get the transient name out of the callables module and delete the transient. Here's a link: https://github.com/Automattic/jetpack/blob/master/sync/class.jetpack-sync-module-callables.php#L7